src/services/solution-is-remote-solution-module/solution-is-remote-solution.module.ts

```diff
@@ -0,0 +1,5 @@
+const httpRegex: RegExp = /^(http|https):\/\//;
```

Comment:
```suggestion
const httpRegex: RegExp = /^(http|https):\/\/.+/i;
```

Ich würde hier noch `ignore-case` als Modifier hinzufügen und sicherstellen, dass die URI im hierarchischen Teil nicht nur `//` enthält.